src/agent/src/unix_main.cpp

```diff
@@ -0,0 +1,89 @@
+#include <cmd_ln_parser.hpp>
+#include <logger.hpp>
+#include <process_options.hpp>
+#include <register.hpp>
+
+#include <unix_daemon.hpp>
+
+#include <chrono>
```

Comment:
Are these two headers necessary? If not, please remove them.

Reply:
Removed